### DISCUSSION TITLE
32bit?

### DISCUSSION THREAD

**Discussion by ElectroLund** [zero-plusplus/vscode-autohotkey-debug#251] [Q&A]
I have a script which uses a 32-bit DLL.  The DLL calls are all failing because this debugger apparently is only using the AutoHotkeyU64.exe runner?  Where can I change it to use the 32 bit?

**Comment by zero-plusplus** [comment]
Are you installing AutoHotkey using the installer?

If so, please change the `runtime` attribute of launch.json to the following.

```
"runtime": "AutoHotkeyU32.exe"
```

If you are not familiar with launch.json, take a look [here](https://code.visualstudio.com/docs/editor/debugging#_launch-configurations).

**Reply by ElectroLund** [reply]
There doesn't appear to be a launch.json that I can find for this extension.  Why not have this and other settings in an extension settings panel?

**Reply by zero-plusplus** [reply]
There is a plan to allow the default value of `runtime` to be changed in settings.json, but not immediately.

VSCode is designed to set debug configuration in launch.json. Therefore you need to learn about [launch.json](https://code.visualstudio.com/docs/editor/debugging#_launch-configurations).
